src/unittest/test.cpp

```diff
@@ -379,6 +379,30 @@ Markdown lol \\|\\|spoiler\\|\\| \\~\\~strikethrough\\~\\~ \\`small \\*code\\* b
 		set_test(SNOWFLAKE, success);
 	}
 
+        { // test snowflake: std::format support
+
+                #if __cplusplus >= 202002L
```

Comment:
DRY,  I'm concerned someone may edit one of these checks and not the other, can we have it purely in one place please, maybe used to set our own define called DPP_HAS_FORMAT

Reply:
Done